client/elife-theme/src/icons/material/Search.js

```diff
@@ -3,6 +3,7 @@ import React from 'react'
 const Search = props => (
   <svg
     height="24"
+    id="search-icon"
```

Comment:
Could we make this `data-test-id`?

Reply:
Well, you're right that I use it only in tests right now but I usually see no harm in adding ids to svg elements, just so I can target them if I need to (e.g. for applying animations).

Reply:
I would probably suggest that we select it via another method than hard coding an `id` into the icon. For example, what if there were multiple search icons on a single page? Using an `id` as a selector for app functionality (rather than tests) may not be the best idea.

Reply:
Highly unlikely that we have multiple search icons on a single page, but I agree with you, I'll change this to `data-test-id`.